std/testing/asserts_test.ts

```diff
@@ -159,6 +160,19 @@ Deno.test("testingNotEquals", function (): void {
   assertEquals(didThrow, true);
 });
 
+Deno.test("testingAssertExists", function (): void {
+  assertExists("Denosaurus");
```

Comment:
Still think we need to have proper coverage for this...  Especially with ["falsey" values](https://developer.mozilla.org/en-US/docs/Glossary/Falsy).

```ts
assertExists(false);
assertExists(0);
assertExists("");
assertExists(-0);
assertExists(0n);
assertExists(NaN);
```